### PR TITLE
Submitting First Exercise!

### PR DESCRIPTION
**Pull request by lyonsa**
Finished turtle exercise. It's a cool spirograph thing.

### CODE REVIEW

**Comment by eah13** [comment]
@lyonsa please include a link to your working site with your pull requests.

**Comment by lyonsa** [comment]
http://lyonsa.github.io/spring2014/2014/01/16/lyonsFirstExercise.html

File: _posts/2014-01-16-lyonsFirstExercise.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+while count<(2*math.pi):
+    if count>0.001:
+        alex.pendown()
+    alex.setpos((R-r)*math.cos(count) + p*math.cos((R-r)*count/r),(R-r)*math.sin(count) - p*math.sin((R-r)*count/r))
+    count += .001
+    
+window.exitonclick()
+```
+
+Here's where I got the formula for a spirograph: [http://www.mathematische-basteleien.de/spirographs.htm](right here)
```
Comment (eah13) [inline _posts/2014-01-16-lyonsFirstExercise.md]: This link is backwards- put the URL in the `()` and the text in the `[]`.  Nice work citing your source :)

**Comment by eah13** [comment]
Nice work! :bear:

**Pull request merged**